content/en/tracing/profiling/_index.md

```diff
@@ -66,25 +66,31 @@ The Datadog Profiler requires [Java Flight Recorder][1]. The Datadog Profiling l
 
 {{% tab "Python" %}}
 
-The Datadog Profiler requires Python 2.7 or above running on a linux OS. Memory profiling will only work on Python 3.5 and above. To begin profiling applications:
+The Datadog Profiler requires Python 2.7 or above running on Linux. Memory profiling will only work on Python 3.5 and above. To begin profiling applications:
 
-1. install `ddtrace`, which contains the both tracing and profiling class files:
+1. install `ddtrace` with the `profile` flavor, which contains both tracing and profiling:
 
     ```shell
-    pip install ddtrace
+    pip install ddtrace[profile]
     ```
 
      **Note**: Profiling is available in the `ddtrace` library in versions above 0.35.
 
 2. Add a valid [Datadog API key][2] in your environment variable: `DD_PROFILING_API_KEY`
 
-3. To automatically profile your code, you can import the `ddtrace.profile.auto module`. As soon as it is imported, it will start catching CPU profiling information on your behalf:
+3. Set `env`, `service` and `version` as Datadog tags in your environment variables.
+
+    ```shell
+    export DD_PROFILING_TAGS=env:<YOUR_ENVIRONMENT>,service:<YOUR_SERVICE>,version:<YOUR_VERSION>
```

Comment:
Just a heads up that as soon as we implement the new standard of DD_ENV, DD_SERVICE, DD_VERSION, we'll want to update this to use those.

Reply:
yes